.github/workflows/pr-build.yml

```diff
@@ -80,6 +80,11 @@ jobs:
           gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
           gpg_password: ${{ secrets.GPG_PASSPHRASE }}
 
+      - name: Dry Run Validator
+        if: ${{ matrix.os != 'windows-latest' }} # Skip dry run validator on windows as it is not possible to build opentelemetry-java on windows
+        run: |
+          ./gradlew testing:validator:run --args='--dry-run-compile true'
```

Comment:
you don't need this to just compile. 

you can use `./gradlew testing:validator:build`

Reply:
Done with the change, thanks